conf/tachyon-env.sh.template

```diff
@@ -58,12 +58,16 @@ CONF_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 export TACHYON_JAVA_OPTS+="
   -Dlog4j.configuration=file:$CONF_DIR/log4j.properties
   -Dtachyon.debug=false
+  -Dtachyon.worker.hierarchystore.level.max=1
```

Comment:
instead of calling level0, level1, level2 etc., how about calling them l0, l1, l2? If this sounds a good idea, please create a JIRA and do in another PR.